ast/ddl.go

```diff
@@ -234,10 +234,6 @@ const (
 	ColumnOptionNotNull
 	ColumnOptionAutoIncrement
 	ColumnOptionDefaultValue
-	ColumnOptionUniq
```

Comment:
Yes, unused according to my whole-project searching... Seems that those constants were moved to constraints. For safety I suggest to take a look since I may not fully understand the architecture.